dataplane/linux/xdp_state.go

```diff
@@ -545,21 +545,21 @@ func (s *xdpIPState) getIPSetMembers(setID string, isSource ipsetsSource) (set.S
 	return getIPSetMembers(s.ipsetIDsToMembers, setID, isSource)
 }
 
-// tryResync performs the resynchronization of the XDP state. It
-// modifies the BPF actions based on the state of XDP on the system
-// and on the desired state. It also repopulates the members cache.
-//
-// This function ensures that after applying the BPF actions, the XDP
-// state will be consistent. Which means making sure that XDP programs
-// are installed in desired interfaces, that they are referencing
-// correct maps, and that maps contain the desired ipsets.
-func (s *xdpIPState) tryResync(common *xdpStateCommon, isSource ipsetsSource) error {
+// tryResync performs reconciles global XDP state with the by using input ipSets
```

Comment:
Please revert this comment change too, you've introduced several errors here and the original comment was pretty good.

Reply:
done, can maybe look at that in a follow on issue.